src/Routing.tsx

```
@@ -1,43 +1,37 @@
-import MigrationWizardPage from './pages/MigrationWizardPage';
+import React, { Suspense } from "react";
+import { Route as RouterRoute, Routes as RouterRoutes } from "react-router-dom";
+import { InvalidObject } from "@redhat-cloud-services/frontend-components/InvalidObject";
+import { Bullseye, Spinner } from "@patternfly/react-core";
+import MigrationAssessmentPage from "./pages/MigrationAssessmentPage";
+import MigrationWizardPage from "./pages/MigrationWizardPage";
 
 interface RouteType {
   path?: string;
-  element: React.ComponentType<Record<string, unknown>>;
+  element: React.ComponentType<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)